openssl/src/hash.rs

```diff
@@ -818,7 +821,10 @@ mod tests {
         }
 
         assert_eq!(MessageDigest::shake_256().block_size(), 136);
+        #[cfg(ossl111)]
         assert_eq!(MessageDigest::shake_256().size(), 32);
+        #[cfg(awslc)]
+        assert_eq!(MessageDigest::shake_128().size(), 0);
```

Comment:
This seems both wrong, and a copy-paste error 😂

Reply:
Ah this one is legitimate mistake to swap the algorithm.